Dockerfile

```
@@ -0,0 +1,33 @@
+FROM node:16.13.0-alpine as build
+
+WORKDIR /app
+
+COPY package.json .yarnrc.yml /app/
+COPY .yarn /app/.yarn
+COPY packages/@uppy/companion /app/packages/@uppy/companion
+
+RUN apk --update add  --virtual native-dep \
+  make gcc g++ python3 libgcc libstdc++ git && \
+  cd /app && corepack yarn workspaces focus @uppy/companion && \
```

Using a subshell so the next commands runs in the same initial directory (in case we add more in the future that cares about the cwd)
```suggestion
  (cd /app && corepack yarn workspaces focus @uppy/companion) && \
```